staging/src/k8s.io/metrics/IMPLEMENTATIONS.md

```diff
@@ -20,5 +20,7 @@ They are listed here for convenience.***
   implementation of the custom metrics API that attempts to support
   arbitrary metrics following a set label and naming scheme.
 
+- [Microsoft Azure Adapter](https://github.com/jsturtevant/azure-k8-metrics-adapter). An implementation of the custom metrics API that allows you to retrieve arbitrary metrics from Azure Monitor.
```

Comment:
The link to the adapter has moved under the Azure org in Github.  Could we change https://github.com/jsturtevant/azure-k8-metrics-adapter to  https://github.com/Azure/azure-k8s-metrics-adapter

@tomkerkhove  or @neolit123  Could we update the link?  Thanks!

Reply:
Done! Sorry for that.